pyiron_base/project/generic.py

```diff
@@ -182,6 +183,17 @@ def data(self):
                 pass
         return self._data
 
+    @property
+    def browser(self):
+        try:
+            from pyiron_gui import ProjectBrowser
```

Comment:
I am not so happy with `pyiron_base` importing `pyiron_gui` that feels like spaghetti code to me. Can not the issue be solved by creating a `Project` class in `pyiron_gui` ?

Reply:
That's what currently is done in the [`activate_gui` function](https://github.com/pyiron/pyiron_gui/blob/3df43717c29e0de4c0d212a004bee9870a5cdc62/pyiron_gui/project/project.py#L20) in the `pyiron_gui` repo - even for all possible `Projects` (base, atomisitics, ...). However, it was requested in the ADIS meeting yesterday, that everything should be possible with a plain `from pyiron(_module) import Project`, i.e. without further imports.

Reply:
It's also a bit weird, because you'd have to do `from pyiron_gui import ProjectAtomisticGui`, `from pyiron_gui import ProjectContribGui`, etc.

Reply:
IMO it's better to break "everything comes from 'from pyiron_x import foo'" than it is to make spaghetti of the code. I just stopped by to correct the attribution of 'ImportAlarm" from myself to Marvin, so I may be missing something. If it's really user pattern vs code structure, I strongly prefer to maintain the atypical use pattern in order to preserve a more sensible codebase. I'm confident we can get a solution that does both in the future.